docs/en/sql-reference/statements/create/view.md

```diff
@@ -66,6 +66,10 @@ A materialized view is implemented as follows: when inserting data to the table
 Materialized views in ClickHouse use **column names** instead of column order during insertion into destination table. If some column names are not present in the `SELECT` query result, ClickHouse uses a default value, even if the column is not [Nullable](../../data-types/nullable.md). A safe practice would be to add aliases for every column when using Materialized views.
 
 Materialized views in ClickHouse are implemented more like insert triggers. If there’s some aggregation in the view query, it’s applied only to the batch of freshly inserted data. Any changes to existing data of source table (like update, delete, drop partition, etc.) does not change the materialized view.
+
+By default if the will be an error during pushing to the materialized view the data will not be inserted into the table to which the materialized view is attached. You can change this by setting `materialized_views_ignore_errors=true` setting for your `INSERT` query.
```

Comment:
This description is unclear to me. Currently, if there is an error on a MV data will be ingested on the landing table (or at least some blocks), other MVs and it might or might not have ingested data on the MV target. The setting seems to be a "ignore any error" and push wherever it's possible.

Reply:
Yeah, without transaction the behavior is undetermined, but since documentation has nothing about "transactions", then I will just write about non deterministic behavior by default and how it can be changed.

@Algunenano Does new documentation paragraphs more clear to you?

Reply:
Yes. Much better. Thanks